tasktiger/__init__.py

```diff
@@ -89,8 +89,12 @@ def __init__(self, connection=None, config=None, setup_structlog=False):
             'DEFAULT_QUEUE': 'default',
 
             # After how many seconds time out on listening on the activity
-            # channel and check for scheduled or expired items.
+            # channel and check for scheduled or expired items.  The batch
+            # timeout will delay the specified seconds after the first message
+            # to wait for additional messages, useful for very active systems.
+            # Appropriate values: 0 >= SELECT_BATCH_TIMEOUT <= SELECT_TIMEOUT
```

Comment:
One more time. This should really read:

`0 <= SELECT_BATCH_TIMEOUT <= SELECT_TIMEOUT`